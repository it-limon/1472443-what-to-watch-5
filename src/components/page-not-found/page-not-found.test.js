import React from "react";
import renderer from "react-test-renderer";
import PageNotFound from "./page-not-found";
import {Route, BrowserRouter} from "react-router-dom";

describe(`Render PageNotFound`, () => {

  it(`Render PageNotFound without link to Main`, () => {
    const tree = renderer
      .create(
        <PageNotFound fromMainPage={true}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Render PageNotFound with link to Main`, () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Route>
            <PageNotFound />
          </Route>
        </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
