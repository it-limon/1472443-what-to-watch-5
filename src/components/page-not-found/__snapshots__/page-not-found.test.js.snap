// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageNotFound Render PageNotFound with link to Main 1`] = `
Array [
  <h1>
    404.
    <br />
    <small>
      Page not found
    </small>
  </h1>,
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main page
  </a>,
]
`;

exports[`Render PageNotFound Render PageNotFound without link to Main 1`] = `
<h1>
  404.
  <br />
  <small>
    Page not found
  </small>
</h1>
`;
